### PR TITLE
feat: discovery entity changes for 1.4 release of sdk

### PR DESCRIPTION
**Pull request by Nandana-NNR**

### CODE REVIEW

File: cypress/TestCases/FireboltCertification/DiscoveryLaunchCold.feature [hunk 1]
```diff
@@ -113,7 +156,68 @@ Feature: Discovery.launch_ColdLaunch
...
-            | Passing random string | launch app with null intent source     | get initialization parameters for null intent     | nullintent source for initialization parameters          |
-            | Passing valid string  | launch app with playback intent source | get initialization parameters for playback intent | playbackintent source for initialization parameters      |
\ No newline at end of file
+    Examples:
+        | Scenario              | Discovery_Launch_Key                   | Call_Parameters_Initialization_With_Context_Key   | Validation_Key_For_Parameters_Initialization_With_Intent |
+        | Passing random string | launch app with null intent source     | get initialization parameters for null intent     | nullintent source for initialization parameters          |
+        | Passing valid string  | launch app with playback intent source | get initialization parameters for playback intent | playbackintent source for initialization parameters      |
+
+    # Below two scenarios are purely added for sdk certification testing and sign off due to limitation in testing discovery launch testcases with current support
```
Comment (neeradanelxsi) [inline cypress/TestCases/FireboltCertification/DiscoveryLaunchCold.feature:164]: below test cases to be removed before merge

**Review by jbigel — APPROVED** [review]

**Review by neeradanelxsi — CHANGES_REQUESTED** [review]
Some of the test cases to be removed before merge. rest all looks okay. Requesting changes so that unnecessary test cases does not get merged

File: cypress/TestCases/FireboltCertification/DiscoveryLaunchCold.feature (hunk 1 above)
Comment (PreethiMaai) [reply cypress/TestCases/FireboltCertification/DiscoveryLaunchCold.feature]: done

**Review by jmeier204 — APPROVED** [review]

**Review by neeradanelxsi — DISMISSED** [review]

**Review by ibrahimshuail — APPROVED** [review]

**Pull request merged**